views/chorals.pug

```diff
@@ -28,5 +23,5 @@ block content
           td= choral.sampleRate
           td= choral.func
           td
-            button.btn.btn-danger.delete-choral-btn(data-choral-id=choral.choralId)
```

Comment:
Not your issue but can you set choral.func to  td= viewHelpers.truncateWithDots(choral.func, 30)